source/includes/platform-payments.md.erb

```diff
@@ -458,34 +470,34 @@ Content-Type: application/json
 Authorization: Basic ...
 
 {
-  "destinationBankAccountId" : "03946a54-6b4f-4bae-897d-6ee2baf0e848",
+  "destinationBankAccountId" : "06ee16ce-def4-4433-80f2-ab35256316e1",
```

Comment:
This, and in other places, it was using a wallet id which is confusing if you ctrl+f it.